scripts/new-upstream-snapshot

```diff
@@ -243,9 +243,14 @@ main() {
         fail "failed to read Source from changelog"
     dist=$(dpkg-parsechangelog --show-field Distribution)
 
-    # if present, pull the '~16.04.x' off of the previous entry.
-    sru_ver_suff=$(echo "$prev_pkg_ver" |
-        sed 's,.*~\([0-9.]*\)[.][0-9]$,~\1.1,')
+    sru_ver_suff=$(distro-info --series ${cur_branch#ubuntu/} --release |
```

Comment:
I don't have distro-info ootb, should this be a dep of this repo then or projects like ua-client?

Reply:
This should probably be an official de of the repo or since we don't have comon setup for each of these scripts, I can add it as dep of this script.